src/components/UserDetailsTooltip/BaseUserDetailsTooltip/index.website.tsx

```diff
@@ -1,56 +1,54 @@
 import Str from 'expensify-common/lib/str';
-import lodashGet from 'lodash/get';
 import React, {useCallback} from 'react';
 import {Text, View} from 'react-native';
-import _ from 'underscore';
 import Avatar from '@components/Avatar';
 import {usePersonalDetails} from '@components/OnyxProvider';
 import Tooltip from '@components/Tooltip';
+import UserDetailsTooltipProps from '@components/UserDetailsTooltip/types';
 import useLocalize from '@hooks/useLocalize';
 import * as LocalePhoneNumber from '@libs/LocalePhoneNumber';
 import * as ReportUtils from '@libs/ReportUtils';
 import * as UserUtils from '@libs/UserUtils';
 import useThemeStyles from '@styles/useThemeStyles';
 import CONST from '@src/CONST';
-import {defaultProps, propTypes} from './userDetailsTooltipPropTypes';
 
-function BaseUserDetailsTooltip(props) {
+function BaseUserDetailsTooltip({accountID, fallbackUserDetails, icon, delegateAccountID, shiftHorizontal, children}: UserDetailsTooltipProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const personalDetails = usePersonalDetails();
 
-    const userDetails = lodashGet(personalDetails, props.accountID, props.fallbackUserDetails);
-    let userDisplayName = ReportUtils.getDisplayNameForParticipant(props.accountID);
-    let userLogin = (userDetails.login || '').trim() && !_.isEqual(userDetails.login, userDetails.displayName) ? Str.removeSMSDomain(userDetails.login) : '';
+    const userDetails = personalDetails[accountID] ?? fallbackUserDetails;
```

Comment:
NAB: I think that to keep code consistent we should add optional chaining here, keeping in mind that later on you're doing it like that. Either this or let's remove optional chaining where it's not necessary.
```suggestion
    const userDetails = personalDetails?.[accountID] ?? fallbackUserDetails;
```